src/components/QuestionOfTheDay/StickLabel.tsx

```diff
@@ -1,16 +1,19 @@
+import { displayFrom } from 'helpers/visibilityClassnames'
 import { useRef, useState } from 'preact/hooks'
 import QuestionOfTheDayLabel from 'icons/QuestionOfTheDayLabel'
 import Tilt from 'react-parallax-tilt'
 import classnames, { inset, position, transforms } from 'classnames/tailwind'
 
 const badgeWrapper = classnames(
   position('absolute'),
+  displayFrom('xs'),
```

Comment:
Why do we need to remove fold, if it worked fine?

Reply:
It's better to hide label on extra small screens, impo